### PR TITLE
Update README with performance benchmark for native OTel logs collec…

### PR DESCRIPTION
**Pull request by aunshc**
Update documentation by adding performance benchmark from https://github.com/splunk/sck-otel/blob/develop/README.md to advertise native otel logs collection

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -265,6 +267,26 @@ There are following known limitations of native OTel logs collection:
   Logs collection with fluentd is still recommended if chart deployed with `autodetect.istio=true`.
 - Journald logs cannot be collected natively by Splunk OTel Collector yet.
 
+### Performance of Splunk Connect for Kubernetes-OpenTelemetry
```
Comment (jodeev) [inline README.md:270]: Performance of native OpenTelemetry logs collection

File: README.md [hunk 2]
```diff
@@ -265,6 +267,26 @@ There are following known limitations of native OTel logs collection:
...
+
+Some configurations used with Splunk Connect for Kubernetes-OpenTelemetry can have an impact on overall performance of log ingestion. The more receivers, processors, exporters and extensions that are added to any of the pipelines, the greater the performance impact.
```
Comment (jodeev) [inline README.md:272]: Some configurations used with Splunk OpenTelemetry Connector for Kubernetes ....

File: README.md [hunk 3]
```diff
@@ -265,6 +267,26 @@ There are following known limitations of native OTel logs collection:
...
+
+Splunk Connect for Kubernetes-OpenTelemetry can exceed the default throughput of HEC. To best address capacity needs, Splunk recommends that you monitor the HEC throughput and back pressure on Splunk Connect for Kubernetes-OpenTelemetry deployments and be prepared to add additional nodes as needed.
```
Comment (jodeev) [inline README.md:274]: Should cover both Splunk platform and LO receivers? and remove reference to SCK-OTel...  Below is correction for Splunk platform.

"When sending data to Splunk, it is possible to exceed the default throughput of HEC. To best address capacity needs, Splunk recommends that you monitor the HEC throughput and back pressure on Splunk OpenTelemetry Connector for Kubernetes deployments.

<Remedition for this TBD - we need to provide guidance on action to take if this is an issue for the customer>

Comment (aunshc) [reply README.md:274]: Since there are no performance comparison numbers between the new chart with fluentd vs the new chart with otel logs collection for LO and the impact of more receivers, we should add that post - I'll continue with the correction for the Splunk platform at the moment as a first iteration of this

File: README.md [hunk 4]
```diff
@@ -265,6 +267,26 @@ There are following known limitations of native OTel logs collection:
...
+
+Here is the summary of performance benchmarks run internally.
```
Comment (jodeev) [inline README.md:276]: Here is the summary of performance benchmarks run internally using the default configuration ... [we need to indicate what this is ]

File: README.md [hunk 5]
```diff
@@ -81,6 +81,8 @@ This distribution currently supports:
   the [`splunk_hec`
   exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/splunkhecexporter).
 
+The helm chart currently utilizes fluentd by default for Kubernetes logs collection and supports an option to use native OpenTelemetry logs collection for higher throughput and performance. [Go to Logs collection section for more details along with performance benchmarks run internally](#logs-collection)
+
```
Comment (tmorrisonsfx) [inline README.md:85]: Suggested change: "The Helm chart currently uses Fluentd by default for Kubernetes logs collection, and supports  an option to use native OpenTelemetry logs collection for higher throughput and performance. See the [logs collection section] for more information, along with performance benchmarks run internally."

For the second sentence, create the hyperlink using only part of the sentence. For example, "[logs collection section]".

File: README.md [hunk 6]
```diff
@@ -265,6 +267,26 @@ There are following known limitations of native OTel logs collection:
   Logs collection with fluentd is still recommended if chart deployed with `autodetect.istio=true`.
 - Journald logs cannot be collected natively by Splunk OTel Collector yet.
 
+### Performance of native OpenTelemetry logs collection
+
+Some configurations used with Splunk Connect for Kubernetes-OpenTelemetry can have an impact on overall performance of log ingestion. The more receivers, processors, exporters and extensions that are added to any of the pipelines, the greater the performance impact.
```
Comment (tmorrisonsfx) [inline README.md:272]: Suggested change: " ... processors, exporters, and extensions that are ..."

File: README.md [hunk 7]
```diff
@@ -265,6 +267,26 @@ There are following known limitations of native OTel logs collection:
...
+
+Splunk Connect for Kubernetes-OpenTelemetry can exceed the default throughput of HEC. To best address capacity needs, Splunk recommends that you monitor the HEC throughput and back pressure on Splunk Connect for Kubernetes-OpenTelemetry deployments and be prepared to add additional nodes as needed.
```
Comment (tmorrisonsfx) [inline README.md:274]: Suggested change: "Splunk Connect for Kubernetes-OpenTelemetry can exceed the default throughput of the The HTTP Event Collector (HEC). To best address capacity needs, monitor the HEC throughput and back pressure on Splunk Connect for Kubernetes-OpenTelemetry deployments and be prepared to add additional nodes as needed."

**Review by tmorrisonsfx — COMMENTED** [review]
I made a few suggestions based on the Splunk Style Guide. Thanks!

File: README.md (hunk 6 above)
Comment (aunshc) [reply README.md:272]: Hi @tmorrisonsfx , thank you for all the suggestions! Can you elaborate on this one more?

Comment (tmorrisonsfx) [reply README.md:272]: Yes, it's just to add a comma after exporters. Sorry for the confusion. Thanks!

File: README.md (hunk 5 above)
Comment (aunshc) [reply README.md:85]: Thanks Traci, just updated it as part of the latest changes

File: README.md [hunk 8]
```diff
@@ -163,9 +165,8 @@ The command removes all the Kubernetes components associated with the chart and
 deletes the release.
 
 ## Advanced Configuration
-
 To fully configure the Helm chart, see the [advanced
-configuration](docs/advanced-configuration.md).
```
Comment (dmitryax) [inline README.md:168]: This is unrelated and seems unnecessary. The current link is working fine.

File: README.md [hunk 9]
```diff
@@ -163,7 +165,6 @@ The command removes all the Kubernetes components associated with the chart and
 deletes the release.
 
 ## Advanced Configuration
-
```
Comment (dmitryax) [inline README.md:166]: Please bring back this line :)

**Review by dmitryax — APPROVED** [review]
LGTM
